packages/malloy-db-bigquery/src/bigquery_connection.ts

```diff
@@ -105,7 +105,10 @@ export class BigQueryConnection extends Connection {
   private temporaryTables = new Map<string, string>();
   private defaultProject;
 
-  private resultCache = new Map<string, MalloyQueryData>();
+  private resultCache = new Map<
```

Comment:
Do we actually want dialect-specific result caching? Not asking for this PR, but generally...